### PR TITLE
Added support for Disk functionality

### PR DESCRIPTION
**Pull request by MuthukumarHelios**
DISK functionality Added
->list Disc, get Disc, Delete Disk

### CODE REVIEW

File: utils/k8s/openebscrd.go [hunk 1]
```diff
@@ -40,3 +40,37 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
...
+
+// DeleteDisk delete disk with diskname
+func (k8s K8S) DeleteDisk(diskName string) error {
+	diskClient := k8s.OpenebsClientSet.OpenebsV1alpha1()
+	err := diskClient.Disks().Delete(diskName, &meta_v1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	return nil
+
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: Lets remove this blank space.

File: utils/k8s/openebscrd.go [hunk 2]
```diff
@@ -40,3 +40,37 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
...
+// GetDisk ... get disk with diskname
+func (k8s K8S) GetDisk(diskName string) (*openebs_v1.Disk, error) {
+
+	diskClient := k8s.OpenebsClientSet.OpenebsV1alpha1()
+	disk, err := diskClient.Disks().Get(diskName, meta_v1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return disk, nil
+
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: Remove this blank space.

File: utils/k8s/openebscrd.go [hunk 3]
```diff
@@ -40,3 +40,37 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// ListDisks  list All disks
+func (k8s K8S) ListDisks() (*openebs_v1.DiskList, error) {
+
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: Remove this blank spcae

**Comment by Akash4927** [comment]
@MuthukumarHelios Please provide a proper pull request message.

File: utils/k8s/openebscrd.go [hunk 4]
```diff
@@ -40,3 +40,33 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
...
+func (k8s K8S) ListDisks() (*openebs_v1.DiskList, error) {
+	diskClient := k8s.OpenebsClientSet.OpenebsV1alpha1()
+	diskList, err := diskClient.Disks().List(meta_v1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return diskList, nil
+}
+
+// GetDisk ... get disk with diskname
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: @MuthukumarHelios 
Can we remove the `...` thing.
It should be following:
`GetDisk  gets disk with diskname.`

File: utils/k8s/openebscrd.go [hunk 5]
```diff
@@ -40,3 +40,33 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
...
+func (k8s K8S) GetDisk(diskName string) (*openebs_v1.Disk, error) {
+	diskClient := k8s.OpenebsClientSet.OpenebsV1alpha1()
+	disk, err := diskClient.Disks().Get(diskName, meta_v1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return disk, nil
+}
+
+// DeleteDisk delete disk with diskname
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: Put it like this:
`// DeleteDisk deletes disk with diskname.`

File: utils/k8s/openebscrd.go [hunk 6]
```diff
@@ -40,3 +40,33 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// ListDisks  list All disks
```
Comment (sonasingh46) [inline utils/k8s/openebscrd.go]: I would prefer following:
`// ListDisks  lists all disks.`

**Pull request merged**